lib/routes/withdraw_funds/withdraw_funds_page.dart

```diff
@@ -135,8 +135,8 @@ class WithdrawFundsPageState extends State<WithdrawFundsPage> {
                   }
                   if (amount > widget.policy.maxValue) {
                     err = texts.withdraw_funds_error_max_value(
-                      acc.currency.format(widget.policy.maxValue + 1),
-                    );
+                        acc.currency.format(widget.policy.maxValue + 1),
+                        acc.currency.format(widget.policy.maxValue + 1));
                   }
```

Comment:
Is this correct? It is duplicated. Also I think its wrong as you can see in the code below, it says minValue where it should be maxValue.

```
  "withdraw_funds_error_max_value": "Must be less than {maxValue}",
  "@withdraw_funds_error_max_value": {
    "placeholders": {
      "minValue": {
        "type": "String",
        "example": "12 345"
      }
    }
  },
```

Reply:
Good point.